lib/watir-webdriver/elements/element.rb

```diff
@@ -125,6 +125,7 @@ def click(*modifiers)
 
           action.perform
         else
+          @element.location_once_scrolled_into_view if driver.capabilities.browser_name == 'MicrosoftEdge'
```

Comment:
Guess that's a bug in EdgeDriver, shouldn't element be automatically scrolled into view?